### ISSUE TITLE
List of users per crp

### ISSUE THREAD

**Issue by carios1usb** [CCAFS/MARLO#976]

**Comment by sebas932** [comment]
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-fc32637c86dfb87633b842e6e00578e08f22e50293b06974958caae4b998fbb9.png)

**Comment by htobon** [comment]
Hi @sebas932 and @cgarcia9106, althoug this sections was finished today. May you please make the following final adjustments?

- [ ]  Whenever you include time (hours, minutes and seconds), you **must** specify the time zone used. I suggest you to include the time zone in the title of the column in parenthesis.
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-7b6a3b8e7c50fc4a1cf38857ccf87c3b74cf215315037a7281182700f698d7b5.png)

- [ ]  Instead of Admin, please rephrase it to "CRP-Admin" and inside the tab, you should say "WLE Administrators"
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-527863de0dc3ab92647af6d20a6cffbd1f7788e3e0d2fb5bea76774504ee7f53.png)

- [ ]  Please delete the ( 's ) at the end of title under all the sections and just add a 's'
Project leaders, Flagship Managers, Program Management Unit, Project Leaders, Project Coordinators, Guests, etc. 
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-8624bf274b37651f24b7802ebc245314dd1c528644324b243b4413922b8da814.png)

- [ ]  Please name the section as "List of users"
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-82bb6a5fff399073da5e0adb23a3f84e89c8cb615402699d9ef26c9421c41b8c.png)

5. Please the ordering feature for the Last login column as it is not properly working:
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-db2ba90f2e6be5df1d6c5e503bf75fb7bb80dc1ec4d7fc3abc194e12b353adf6.png)

**Comment by sebas932** [comment]
@cgarcia9106 please fix the last one item about role names

**Comment by sebas932** [comment]
Done
![image.png](https://overvio-usercontent.s3.eu-central-1.amazonaws.com/image-552e7f568b7ecc2e0b893b1d08465e177eec99f794c91e90f6ea1ce682a7572e.png)
